first-timers.md

```diff
@@ -0,0 +1,8 @@
+# How to Contribute to SendGrid Open Source Projects  
+
+##To get started, please select a repo:   
```

Comment:
This line should be `## To get started, please select a repo:`. It misses a white space befotre `To` word :).